mica/archive/obsid_archive.py

```diff
@@ -753,7 +766,15 @@ def update(self):
                         current_status['quality'] = 'X'
                 else:
                     # if there are no science_2 entries at all for the obsid
-                    # try an update anyway
+                    # see if it is discarded
+                    with Ska.DBI.DBI(dbi='sybase', server='sqlsao', database='axafocat') as db:
+                        target_status = db.fetchone(
+                            "select status from target where obsid = {}".format(obs['obsid']))
+                        if target_status['status'] == 'discarded':
+                            logger.info("Skipping {}, obsid 'discarded'".format(
```

Comment:
Also, skip obsids that are now 'discarded' as they are never going to have 'default'/released products in the archive.